lib/logger.js

```diff
@@ -96,7 +96,10 @@ logger.banner = function () {
 	var info = appc.pkginfo.package(module, 'version', 'about');
 	if (bannerEnabled) {
 		logger.log(info.about.name.cyan.bold + ', CLI version ' + info.version + (logger.activeSdk ? ', Titanium SDK version ' + logger.activeSdk : '') + '\n' + info.about.copyright);
-		logger.log('\n' + __('Please report bugs to %s', 'https://github.com/appcelerator/titanium_mobile/issues'.cyan) + '\n');
+		logger.log('\n' + __('- Please report bugs here: %s', 'https://github.com/tidev/titanium_mobile/issues'.cyan));
+		logger.log(__('- Remember! Titanium is a community supported open-source project and relies on donations for on-going development.'));
+		logger.log(__('- Support the project here: %s', 'https://liberapay.com/tidev'.cyan));
```

Comment:
I profoundly disagree with putting any mention of donations in the CLI output. I've already said time and time again, Titanium cannot sustain off donations. Titanium is open source. It is free. The only way Titanium will survive is if we get people to contribute. You set a precedence when you ask for donations to fund development. It's unfair to all the contributors that work on Titanium for free. Soon, nobody will want to contribute for free.